src/sentry/runner/commands/devserver.py

```diff
@@ -14,9 +14,9 @@
 _DEV_METRICS_INDEXER_ARGS = [
     # We don't want to burn laptop CPU while idle, but do want for
     # metrics to be ingested with lowest latency possible.
-    "--max-batch-time-ms",
+    "--max-msg-batch-time-ms",
     "10000",
-    "--max-batch-size",
+    "--max-msg-batch-size",
```

Comment:
@untitaker not sure if we also want to set `--max-parallel*` options here:

https://github.com/getsentry/sentry/blob/35be0864b883a7877e941d0acd174f8358bf8636/src/sentry/runner/commands/run.py#L638-L639